docs/design/autoware-interfaces/ad-api/types/autoware_ad_api_msgs/msg/driving_state.md

```diff
@@ -0,0 +1,16 @@
+# autoware_ad_api_msgs/msg/DrivingState
+
+## Definition
+
+```txt
+uint16 UNKNOWN = 0
+uint16 PREPARING = 1
```

Comment:
Which is correct, PREPARING or NOT_READY?
And it seems there is not `UNKNOWN` in the diagram.

Reply:
Fixed https://github.com/autowarefoundation/autoware-documentation/pull/115/commits/9da5318b7a2b08d1fbde4a2c757f2bdbba4b9617. `UNKNOWN` means that it is not in any state (e.g. when the state has never been subscribed).